kernel/Kconfig

```diff
@@ -1020,6 +1020,13 @@ config THREAD_LOCAL_STORAGE
 	help
 	  This option enables thread local storage (TLS) support in kernel.
 
+config KERNEL_WHOLE_ARCHIVE
+	bool "Link Kernel Library as whole archive"
+	default n
```

Comment:
`default n` can be omitted.

Reply:
Good point, I've now removed it in-favor of the implicit default.